templates/base.html

```diff
@@ -83,14 +58,16 @@
             </li>
           </ul>
           {% if user.is_authenticated %}
+          <a id class="nav-link me-sm-2" href="/accounts/edit_profile">Edit Profile</a>
           <a id class="nav-link me-sm-2" href="/accounts/logout">Logout</a>
           {% else %}
           <a id class="nav-link me-sm-2" href="/accounts/login">Login</a>
           {% endif %}
         </div>
-        </div>
-      </nav>
-      <div id="outer-container">{%block content%}{%endblock%}</div>
-    </main>
-  </body>
-</html>
+      </div>
+    </nav>
+    <div id="outer-container">{%block content%}{%endblock%}</div>
+  </main>
+</body>
+
+</html>
```

Comment:
missing line

Reply:
In html files the formating is without a new line I think